docs/using-radicle/running-a-seed-node.md

```diff
@@ -3,5 +3,92 @@ id: running-a-seed-node
 title: Running a seed node
 ---
 
-This is a link to [another document.](doc3.md)  
-This is a link to an [external page.](http://www.example.com)
+To improve data availability, participants in the network can choose to act as
+seeds. This is similar in concept to a pub in Secure Scuttlebutt: seeds are
+always-on nodes that automatically track discovered projects, thereby
+increasing the availability of these projects on the network.
+
+To build and run a seed node, you'll have to install some prerequisites on your
+machine first:
+
+  - [Rust toolchain][rust]
+  - [yarn][yarn]
+
+Next, clone and set up the `radicle-bins` repository:
+
+    git clone https://github.com/radicle-dev/radicle-bins.git
+    cd radicle-bins
+    git submodule update --init --recursive
```

Comment:
This won't be needed anymore.

Reply:
https://github.com/radicle-dev/radicle-docs/pull/13/commits/be4397dd8de5f2250f810701920bb4277d307577